Content.Shared/SimpleStation14/CCVar/CCVars.cs

```diff
@@ -16,4 +16,17 @@ public sealed class SimpleStationCCVars
     /// </summary>
     public static readonly CVarDef<List<string>> AnnouncerBlacklist =
         CVarDef.Create("game.announcer.blacklist", new List<string>(), CVar.SERVERONLY);
+
+
+    /// <summary>
+    ///     Whether or not respawning is enabled.
+    /// </summary>
+    public static readonly CVarDef<bool> RespawnEnabled =
+        CVarDef.Create("respawn.enabled", true, CVar.SERVER | CVar.REPLICATED);
```

Comment:
```suggestion
        CVarDef.Create("respawn.enabled", false, CVar.SERVER | CVar.REPLICATED);
```
We can merge this in and then continue discussing it.
Also, probably best for most servers anyway, with turning it on being the exception :P